code/controllers/configuration/configuration.dm

```diff
@@ -417,4 +417,4 @@ Example config:
 
 //Message admins when you can.
 /datum/controller/configuration/proc/DelayedMessageAdmins(text)
-	addtimer(CALLBACK(GLOBAL_PROC, /proc/message_admins, text), 0)
+	addtimer(CALLBACK(GLOBAL_PROC, /proc/message_admins, text), 0, TIMER_UNIQUE)
```

Comment:
this means that you can't send 2 delayed admin messages at once.

Reply:
Callback args are considered in timer unique